3.0.0.dev3 sparse run (wordcount) - Flux parsing error

sparse run

```
Cleaning from prior builds...
Creating topology Uber-JAR...
Uber-JAR created: /home/darkless/projects/sparse_testing/test/wordcount/_build/wordcount-0.0.1-SNAPSHOT-standalone.jar
Removing _resources temporary directory...done
[localhost] local: storm jar /home/darkless/projects/sparse_testing/test/wordcount/_build/wordcount-0.0.1-SNAPSHOT-standalone.jar org.apache.storm.flux.Flux --local --no-splash --sleep 9223372036854775807 /tmp/tmpKADN7C.yaml
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/opt/apache-storm-1.0.1/lib/log4j-slf4j-impl-2.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/darkless/projects/sparse_testing/test/wordcount/_build/wordcount-0.0.1-SNAPSHOT-standalone.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
Running: java -client -Ddaemon.name= -Dstorm.options= -Dstorm.home=/opt/apache-storm-1.0.1 -Dstorm.log.dir=/opt/apache-storm-1.0.1/logs -Djava.library.path=/usr/local/lib:/opt/local/lib:/usr/lib -Dstorm.conf.file= -cp /opt/apache-storm-1.0.1/lib/reflectasm-1.10.1.jar:/opt/apache-storm-1.0.1/lib/kryo-3.0.3.jar:/opt/apache-storm-1.0.1/lib/log4j-over-slf4j-1.6.6.jar:/opt/apache-storm-1.0.1/lib/clojure-1.7.0.jar:/opt/apache-storm-1.0.1/lib/log4j-slf4j-impl-2.1.jar:/opt/apache-storm-1.0.1/lib/servlet-api-2.5.jar:/opt/apache-storm-1.0.1/lib/disruptor-3.3.2.jar:/opt/apache-storm-1.0.1/lib/objenesis-2.1.jar:/opt/apache-storm-1.0.1/lib/storm-core-1.0.1.jar:/opt/apache-storm-1.0.1/lib/slf4j-api-1.7.7.jar:/opt/apache-storm-1.0.1/lib/storm-rename-hack-1.0.1.jar:/opt/apache-storm-1.0.1/lib/log4j-api-2.1.jar:/opt/apache-storm-1.0.1/lib/log4j-core-2.1.jar:/opt/apache-storm-1.0.1/lib/minlog-1.3.0.jar:/opt/apache-storm-1.0.1/lib/asm-5.0.3.jar:/home/darkless/projects/sparse_testing/test/wordcount/_build/wordcount-0.0.1-SNAPSHOT-standalone.jar:/opt/apache-storm-1.0.1/conf:/opt/apache-storm-1.0.1/bin -Dstorm.jar=/home/darkless/projects/sparse_testing/test/wordcount/_build/wordcount-0.0.1-SNAPSHOT-standalone.jar org.apache.storm.flux.Flux --local --no-splash --sleep 9223372036854775807 /tmp/tmpKADN7C.yaml
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/opt/apache-storm-1.0.1/lib/log4j-slf4j-impl-2.1.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/home/darkless/projects/sparse_testing/test/wordcount/_build/wordcount-0.0.1-SNAPSHOT-standalone.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.apache.logging.slf4j.Log4jLoggerFactory]
Parsing file: /tmp/tmpKADN7C.yaml
401  [main] INFO  o.a.s.f.p.FluxParser - loading YAML from input stream...
402  [main] INFO  o.a.s.f.p.FluxParser - Not performing property substitution.
402  [main] INFO  o.a.s.f.p.FluxParser - Not performing environment variable substitution.
Exception in thread "main" Can't construct a java object for tag:yaml.org,2002:org.apache.storm.flux.model.TopologyDef; exception=Cannot create property=name for JavaBean=org.apache.storm.flux.model.TopologyDef@7d632866; null; Can't construct a java object for tag:yaml.org,2002:python/unicode; exception=Class not found: python/unicode;  in 'string', line 8, column 7:
    name: !!python/unicode 'wordcount'
          ^
 in 'string', line 1, column 1:
    bolts:
    ^

	at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.construct(Constructor.java:333)
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:182)
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructDocument(BaseConstructor.java:141)
	at org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:127)
	at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:481)
	at org.yaml.snakeyaml.Yaml.load(Yaml.java:400)
	at org.apache.storm.flux.parser.FluxParser.loadYaml(FluxParser.java:121)
	at org.apache.storm.flux.parser.FluxParser.parseInputStream(FluxParser.java:75)
	at org.apache.storm.flux.parser.FluxParser.parseFile(FluxParser.java:49)
	at org.apache.storm.flux.Flux.runCli(Flux.java:145)
	at org.apache.storm.flux.Flux.main(Flux.java:98)
Caused by: org.yaml.snakeyaml.error.YAMLException: Cannot create property=name for JavaBean=org.apache.storm.flux.model.TopologyDef@7d632866; null; Can't construct a java object for tag:yaml.org,2002:python/unicode; exception=Class not found: python/unicode;  in 'string', line 8, column 7:
    name: !!python/unicode 'wordcount'
          ^
	at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.constructJavaBean2ndStep(Constructor.java:299)
	at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.construct(Constructor.java:189)
	at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.construct(Constructor.java:331)
	... 10 more
Caused by: Can't construct a java object for tag:yaml.org,2002:python/unicode; exception=Class not found: python/unicode
 in 'string', line 8, column 7:
    name: !!python/unicode 'wordcount'
          ^

	at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.construct(Constructor.java:333)
	at org.yaml.snakeyaml.constructor.BaseConstructor.constructObject(BaseConstructor.java:182)
	at org.yaml.snakeyaml.constructor.Constructor$ConstructMapping.constructJavaBean2ndStep(Constructor.java:296)
	... 12 more
Caused by: org.yaml.snakeyaml.error.YAMLException: Class not found: python/unicode
	at org.yaml.snakeyaml.constructor.Constructor.getClassForNode(Constructor.java:633)
	at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.getConstructor(Constructor.java:321)
	at org.yaml.snakeyaml.constructor.Constructor$ConstructYamlObject.construct(Constructor.java:331)
	... 14 more

Fatal error: local() encountered an error (return code 1) while executing 'storm jar /home/darkless/projects/sparse_testing/test/wordcount/_build/wordcount-0.0.1-SNAPSHOT-standalone.jar org.apache.storm.flux.Flux --local --no-splash --sleep 9223372036854775807 /tmp/tmpKADN7C.yaml'

Aborting.
```


    cat /tmp/tmpKADN7C.yaml

```
bolts:
- className: org.apache.storm.flux.wrappers.bolts.FluxShellBolt
  constructorArgs:
  - [streamparse_run, bolts.wordcount.WordCountBolt]
  - [word, count]
  id: count_bolt
config: {topology.acker.executors: 2, topology.debug: false, topology.workers: 2}
name: !!python/unicode 'wordcount'
spouts:
- className: org.apache.storm.flux.wrappers.spouts.FluxShellSpout
  constructorArgs:
  - [streamparse_run, spouts.words.WordSpout]
  - [word]
  id: word_spout
streams:
- from: word_spout
  grouping:
    args: [word]
    streamId: default
    type: FIELDS
  to: count_bolt
```

--- a/streamparse/cli/run.py
+++ b/streamparse/cli/run.py
@@ -48,7 +48,7 @@ def run_local_topology(name=None, time=0, workers=2, ackers=2, options=None,
     with NamedTemporaryFile(suffix='.yaml', delete=False) as yaml_file:
         topology_flux_dict = topology_class.to_flux_dict(name)
         topology_flux_dict['config'] = storm_options
-        yaml.dump(topology_flux_dict, yaml_file)
+        yaml.safe_dump(topology_flux_dict, yaml_file)
         cmd = ('storm jar {jar} org.apache.storm.flux.Flux --local --no-splash '
                '--sleep {time} {yaml}'.format(jar=topology_jar,
                                               time=time,

